src/algorithms/tracking/AmbiguitySolver.cc

```diff
@@ -0,0 +1,287 @@
+// Created by Minjung Kim (minjung.kim@lbl.gov)
+
+#include "AmbiguitySolver.h"
+#include "AmbiguitySolverConfig.h"
+
+#include "Acts/AmbiguityResolution/GreedyAmbiguityResolution.hpp"
+#if EDM4EIC_VERSION_MAJOR >= 5
```

Comment:
Instead of continuing to use preprocessor directives, we should discuss upgrading the requirement in CMakeLists.txt to `find_package(EDM4EIC 5.0 REQUIRED)`. @veprbl

Reply:
Just assume `EDM4EIC_VERSION_MAJOR >= 5` is true. We'll remove EDM4eic 4 support soon.

Reply:
Filed https://github.com/eic/EICrecon/issues/1389 to track this